app/controllers/api/account.php

```diff
@@ -705,6 +793,86 @@
         ;
     });
 
+App::get('/v1/account/identities')
+    ->desc('List Identities')
+    ->groups(['api', 'account'])
+    ->label('scope', 'account')
+    ->label('usage.metric', 'users.{scope}.requests.read')
+    ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_JWT])
+    ->label('sdk.namespace', 'account')
+    ->label('sdk.method', 'listConnected')
```

Comment:
`listIdentities` makes more sense. Have we considered `getIdentity` also?

Reply:
👆 I agree, `listIdenties` abides more by our conventions

Reply:
This was leftover from when we called it connected accounts. Thanks for catching it!